test/mixin/download/download_file_on_mobile_mixin_test.dart

```diff
@@ -0,0 +1,276 @@
+import 'dart:io';
+
+import 'package:dartz/dartz.dart' hide State, OpenFile, id;
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/presentation/model/chat/downloading_state_presentation_model.dart';
+import 'package:fluffychat/utils/exception/downloading_exception.dart';
+import 'package:fluffychat/utils/manager/download_manager/download_file_state.dart';
+import 'package:fluffychat/utils/manager/download_manager/download_manager.dart';
+import 'package:fluffychat/widgets/mixins/download_file_on_mobile_mixin.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:matrix/matrix.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'download_file_on_mobile_mixin_test.mocks.dart';
+
+const fakeEventId = "fakeEventId";
+const fakeFilename = "fakeFilename";
+
+class MockRoom extends Mock implements Room {
```

Comment:
Create a file that can be reused?

Reply:
IMO for mocking up rooms and events, you can take a look at `message_content_builder_mixin_test.dart`: https://github.com/linagora/twake-on-matrix/blob/main/test/pages/chat/events/message/message_content_builder_mixin_test.dart